soc/nxp/imxrt/imxrt11xx/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ zephyr_include_directories(.)
 if(CONFIG_MEMC_MCUX_FLEXSPI)
   zephyr_sources(flexspi.c)
   if(CONFIG_FLASH_MCUX_FLEXSPI_XIP)
-    zephyr_code_relocate(FILES flexspi.c LOCATION ITCM_TEXT)
+    zephyr_code_relocate(FILES flexspi.c LOCATION ${CONFIG_FLASH_MCUX_FLEXSPI_XIP_MEM})
```

Comment:
Aren't you missing `_TEXT`?

Reply:
I am, this is likely just dropping it in the `ITCM` section. Let me update this